### ISSUE TITLE
Unable to detect numbers

### ISSUE THREAD

**Issue by skywo1f** [GokulNC/DarkNet-Classifier-LeNet-MNIST#2]
Went through the steps, trained network. Showed no issues, loss went down, accuracy went up. Still, when I run the neural network, it is unable to find numbers on a page

**Comment by GokulNC** [comment]
MNIST dataset is a digit "classification" dataset, not "detection" dataset.
And this tutorial is for training a classifier given a image of a digit, not detection.

For digits detection and recognition, please use this repo:
https://github.com/MrEliptik/YOLOv3_digits

**Comment by skywo1f** [comment]
Ill try it out, thanks!

**Comment by skywo1f** [comment]
Unfortunately that repo did not work for me. Firstly it is heavily deprecated, relying on old tensorflow and keras. I was able to update it to run, but the instructions were not clear enough. I can get it to train, but it creates no trained weights file afterwards. Finally, there is no "issues" page for it and the repo it was forked from seems to be lacking any form of support.
